### PR TITLE
setup prisma: schema, seeder and scripts

### PR DESCRIPTION
**Pull request by lebalz**
WIP: Setup prisma schema

- [x] create first draft of a schema with users, groups and documents
- [x] add seeder
- [x] add docs about .env
- [x] add db documentation plugin

### CODE REVIEW

**Comment by lebalz** [comment]
Hi @SilasBerger 

This PR seems fine to merge - please have a look ;)

File: db/scripts/02_seed_db.sql
Comment (SilasBerger) [inline db/scripts/02_seed_db.sql:1]: Feel free to remove `db/scripts/00_purge_teaching_website_db.sql` as well. I don't think we need this anymore with the new setup: When using  Docker, we can either recreate the DB using `yarn db:recreate`, or we can go one setup further and nuke the entire container including its image. This is done with `scripts/purge_dev_services.sh`.

On that note, in `.idea/runConfigurations`, you could also remove `clean_dev_db.xml` and `seed_dev_db.xml`.

File: README.md [hunk 1]
```diff
@@ -1,18 +1,150 @@
 # Teaching Website Backend
 The backend for my teaching website.
```
Comment (SilasBerger) [inline README.md:2]:
```suggestion
The backend for our teaching website.
```

😜

File: src/index.ts [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
 
+app.use(express.static(path.join(__dirname,'..', 'docs')));
+
 app.get('/', async (req, res) => {
-  // res.json({message: "Hello, world!"})
+    res.json({message: "Hello, world!"})
 });
 
-prisma.messages.findMany().then((res: any) => console.log(res));
+// prisma.messages.findMany().then((res: any) => console.log(res));
```
Comment (SilasBerger) [inline src/index.ts:17]: Feel free to remove this line entirely. I originally just added that to check whether Prisma works.

**Comment by SilasBerger** [comment]
This looks great, thanks!

I am not too familiar with Prisma yet, so I didn't go into much detail on these parts. The schema makes sense; imo we can go with that and take it from there :).

I added a few small cleanup comments, otherwise ready to merge as far as I'm concerned.

**Comment by lebalz** [comment]
Thanks @SilasBerger for the feedback

**Pull request merged**